test/addons-abi/test_buffer/test_buffer.cc

```diff
@@ -1,4 +1,4 @@
-#include <string.h>
+﻿#include <string.h>
```

Comment:
I think this is a bug in VS 2017, which Taylor and I have started using more lately. VS 2017 apparently always adds a UTF-8 byte order mark to a source file when editing it even if there are no non-ASCII characters.

I think technically the BOM is required if there are non-ASCII characters in the file, and optional if the file is all ASCII. But VS shouldn't add the BOM when it wasn't there before and isn't necessary.